packages/components/table/src/table/style-helper.ts

```diff
@@ -40,6 +41,7 @@ function useStyle<T extends DefaultRow>(
   const scrollbarViewStyle = {
     display: 'inline-block',
     verticalAlign: 'middle',
+    height: '100%',
```

Comment:
Thanks you for your contribution.

I feel this isn't a good idea, for scrolling components, the `el-scrollbar__view` and `el-scrollbar__wrap` shouldn't have the same height. While it doesn't seem to have any impact now, there could be potential problems.